devops/terraform/bootstrap.sh

```diff
@@ -38,29 +38,67 @@ if [ -n "${ARM_CLIENT_ID:-}" ]; then
 else
     USER_OBJECT_ID=$(az ad signed-in-user show --query id --output tsv)
 fi
+
+az role assignment create --assignee "$USER_OBJECT_ID" \
```

Comment:
Did you work out why this is required? I thought the object ID that creates the storage account should have sufficient permissions?